tests/async.test.tsx

```diff
@@ -498,7 +498,7 @@ it('set promise atom value on write (#304)', async () => {
     set(
       countAtom,
       Promise.resolve(get(countAtom)).then(
-        (c) => new Promise((r) => setTimeout(() => r(c + 1), 500))
+        (c) => new Promise((r) => setTimeout(() => r(c + 1), 1))
```

Comment:
not sure how this works. do they use the fake timer by default?

Reply:
Can you elaborate?

Reply:
In my experience with the previous one, this doesn't pass CI.
So, we would like to a) see if it passes, and b) know why.

Reply:
You need to it forward then :)